src/fontra/views/editor/editor.html

```diff
@@ -47,16 +48,35 @@
                 <!-- content will be filled in dynamically -->
               </div>
               <div id="zoom-tools" class="tools-item">
-                <div class="tool-button" data-tool="zoom-out">
+                <div
+                  class="tool-button"
+                  data-tool="zoom-out"
+                  data-tooltip="zoom-out"
```

Comment:
General comment: I would make the tooltip "normal language", in this case "Zoom out". The `"zoom-out"` string is an identifier, good for the machine, less so for humans.